### PR TITLE
Release note updates

### PR DESCRIPTION
**Pull request by rjeberhard**
Just a placeholder guess on the 3.1.0 release date. Updates for release notes, including descriptions of changes. I'll add the other changes as we commit them (OpenShift Dockerfile change, stuck Terminating pod fix, etc.)

### CODE REVIEW

**Review by ddsharpe — APPROVED** [review]

File: docs-source/content/release-notes.md [hunk 1]
```diff
@@ -30,6 +30,27 @@ draft: false
...
+#### Operator 3.1.0
+
+* Updated [default resource and Java options](faq/resource-settings.md) ([#1775](https://github.com/oracle/weblogic-kubernetes-operator/pull/1775)).
+* Introspection failures are logged to the operator's log ([#1787](https://github.com/oracle/weblogic-kubernetes-operator/pull/1787)).
+* Mirror introspector log to a rotating file in the log home ([#1827](https://github.com/oracle/weblogic-kubernetes-operator/pull/1827)).
+* Reflect introspector status to domain status ([#1832](https://github.com/oracle/weblogic-kubernetes-operator/pull/1832)).
+* Ensure operator detects pod state changes even when watch events are not delivered ([#1811](https://github.com/oracle/weblogic-kubernetes-operator/pull/1811)).
+* Support configurable model home ([#1828](https://github.com/oracle/weblogic-kubernetes-operator/pull/1828)).
+* [Namespace management enhancements](faq/namespace-management.md) ([#1860](https://github.com/oracle/weblogic-kubernetes-operator/pull/1860)).
+* Limit concurrent pod shutdown during cluster shrink ([#1892](https://github.com/oracle/weblogic-kubernetes-operator/pull/1892)).
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:42]: shutdown -> shut down (shut down (v), shutdown (n) )
cluster shrink -> odd wording ( I see that these statements come from JIRA titles, which are not always written well. Should we keep them with their original wording or fix them?)

File: docs-source/content/release-notes.md [hunk 2]
```diff
@@ -30,6 +30,27 @@ draft: false
...
 ### Change log
 
+#### Operator 3.1.0
+
+* Updated [default resource and Java options](faq/resource-settings.md) ([#1775](https://github.com/oracle/weblogic-kubernetes-operator/pull/1775)).
+* Introspection failures are logged to the operator's log ([#1787](https://github.com/oracle/weblogic-kubernetes-operator/pull/1787)).
+* Mirror introspector log to a rotating file in the log home ([#1827](https://github.com/oracle/weblogic-kubernetes-operator/pull/1827)).
+* Reflect introspector status to domain status ([#1832](https://github.com/oracle/weblogic-kubernetes-operator/pull/1832)).
+* Ensure operator detects pod state changes even when watch events are not delivered ([#1811](https://github.com/oracle/weblogic-kubernetes-operator/pull/1811)).
+* Support configurable model home ([#1828](https://github.com/oracle/weblogic-kubernetes-operator/pull/1828)).
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:40]: Support configurable model home -> Support configurable WDT model home

**Review by rosemarymarano — COMMENTED** [review]
Some comments here. I'm not sure of the answer for this from your notes, but do you need input for other docs that need operator version updates, such as the Quick Start, Get images 
https://oracle.github.io/weblogic-kubernetes-operator/quickstart/get-images/
and Install the operator and load balancer 
https://oracle.github.io/weblogic-kubernetes-operator/quickstart/install/
and others?

**Comment by rjeberhard** [comment]
@rosemarymarano,
> I'm not sure of the answer for this from your notes, but do you need input for other docs that need operator version updates, such as the Quick Start, Get images

I believe I caught all of those. I did a find-in-files over the entire codebase.

**Comment by rosemarymarano** [comment]
Hmmm. Maybe you did get them all but I see only 5 changed files in this PR. Perhaps, you changed the others (such as the Quick Start, "Get Images" and and "Install the operator and load balancer" in a previous PR? I'll check again.

**Comment by rjeberhard** [comment]
Yes, I think I got the other files "earlier."

**Comment by rosemarymarano** [comment]
I pulled your branch (release-notes-310) and all the required changes were there, except one (which I see in this PR, but have no idea why it's not showing up in the branch). On the Home page,  weblogic-kubernetes-operator\docs-source\content\_index.md, I see this text, "This release was published on September ?, 2020." I don't see your updated text, "This release was published on November 10, 2020." Not sure why but I thought I should mention it. Other than that, everything is fine.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**